Dockerfile

```diff
@@ -7,20 +7,25 @@ EXPOSE 7778
 
 ## install pipcook dependencies
 
-RUN apt-get update && echo 'Y' | apt install software-properties-common && echo '\n' | add-apt-repository ppa:deadsnakes/ppa \
-  && echo 'Y' | apt install python3.7 && echo 'Y' | apt-get install python3-setuptools && echo 'Y' | apt-get install curl \
-  && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3.7 get-pip.py && ln -s /usr/bin/python3.7 /usr/bin/python \
-  && pip install torch torchvision && pip install opencv-python && echo 'Y' | apt install git && echo 'Y' | apt install build-essential \
-  && echo 'Y' | apt-get install python3.7-dev && pip install 'git+https://github.com/facebookresearch/fvcore' && pip install cython \
+RUN apt-get update \
+  && apt-get --no-install-recommends install -y apt-utils ca-certificates software-properties-common \
+  && echo '\n' | add-apt-repository ppa:deadsnakes/ppa \
+  && apt-get --no-install-recommends install -y python3.7 curl python3-setuptools \
+  && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
+  && python3.7 get-pip.py && ln -s /usr/bin/python3.7 /usr/bin/python \
+  && pip install torch torchvision && pip install opencv-python \
+  && apt-get --no-install-recommends install git build-essential python3.7-dev \
+  && pip install 'git+https://github.com/facebookresearch/fvcore' \
+  && pip install cython \
   && pip install 'git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI' \
   && pip install 'git+https://github.com/facebookresearch/detectron2.git' \
-  && echo 'Y' | apt-get install -y libsm6 libxext6 libxrender-dev
```

Comment:
Keep this print, please :)

Reply:
here "-y" flag is already for that , and provide "--assume-yes" automatically